en-ar/jw300-tedtalk-baseline/.ipynb_checkpoints/starter_notebook (1)-checkpoint.ipynb

```diff
@@ -0,0 +1,1370 @@
+{
```

Comment:
<p>We shouldn't commit anything that's in <code>/.ipynb_checkpoints/</code>  - would you mind removing them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>